docs/develop/go/test-suites.mdx

```diff
@@ -283,6 +286,102 @@ Learn to skip time manually in the SDK of your choice.
 
 Learn to skip time in Activities in the SDK of your choice.
 
+## Testing Nexus Operations {#test-nexus}
```

Comment:
This should be renamed `Mock and override Nexus Operations` and moved below `Mock and override Activities`.